src/flat-button.jsx

```diff
@@ -222,9 +283,9 @@ const FlatButton = React.createClass({
     this.props.onMouseLeave(e);
   },
 
-  _handleTouchStart(e) {
+  _handleTouchTap(e) {
     this.setState({touch: true});
```

Comment:
Ok, so that means adding it enhancedButton, since thats what handles those events. Should I also add the onTouchEnd?